src/hud_elements.cpp

```diff
@@ -250,22 +250,20 @@ void HudElements::gpu_stats(){
                 ImGui::PopFont();
             }
 
-            if (HUDElements.vendorID == 0x1002 || HUDElements.vendorID == 0x10de){
-                if (HUDElements.params->enabled[OVERLAY_PARAM_ENABLED_gpu_fan] && cpuStats.cpu_type != "APU"){
-                    ImguiNextColumnOrNewRow();
-                    right_aligned_text(text_color, HUDElements.ralign_width, "%i", gpu->metrics.fan_speed);
+            if (HUDElements.params->enabled[OVERLAY_PARAM_ENABLED_gpu_fan] && cpuStats.cpu_type != "APU"){
```

Comment:
`cpuStats.cpu_type` should be deprecated in favor of `gpu->is_apu()`.
Only AMD is being set as APU atm so intel needs to be hooked up there

Reply:
cpu_type is completetly unused, even for amd

Reply:
and do you mean it needs changing from `cpuStats.cpu_type != "APU"` to `!gpu->is_apu()`?

Reply:
done, i think?

Reply:
and I don't know currently how to detect if gpu is an integrated one, so that will have to wait

Reply:
Yes it should be changed to `!gpu->is_apu()`

Reply:
I changed it already